proto/v1/vulnerability.proto

```diff
@@ -94,6 +94,7 @@ message Vulnerability {
   }
 
   // ID 4 is unusable due to a proto conflict.
+  reserved 4;
```

Comment:
Please move this to the top of the message for easier visibility. We eventually would want all reserved proto fields in the same place.